documentation/examples.md

```diff
@@ -190,6 +199,11 @@ We can then use the macros as such:
 ```javascript
 D("example.com", REG_NONE, DnsProvider(DSP_R53_MAIN),
     FASTMAIL_MX,
-    FASTMAIL_DKIM('example.com')
+    FASTMAIL_DKIM("example.com")
 )
 ```
+{% endcode %}
+
+### More advanced examples
+
+See the [Code Tricks](advanced-features/code-tricks) page.
```

Comment:
@tlimoncelli this link is incorrect, this should be `code-tricks.md`.

```diff
-See the [Code Tricks](advanced-features/code-tricks) page.
+See the [Code Tricks](code-tricks.md) page.
```

_https://docs.dnscontrol.org/getting-started/examples#more-advanced-examples_